dump/mapping/prepbufr_adpupa.yaml

```diff
@@ -0,0 +1,284 @@
+# (C) Copyright 2025 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache License Version 2.0
+# http://www.apache.org/licenses/LICENSE-2.0.
+
+bufr:
+  group_by_variable: prepbufrDataLevelCategory
+  subsets:
+    - ADPUPA
+
+  variables:
+    # ObsType
+    observationType:
+      query: "*/TYP"
+
+    # MetaData
+    timestamp:
+      timeoffset:
+        timeOffset: "*/PRSLEVEL/DRFTINFO/HRDR"
+        transforms:
+          - scale: 3600
+        referenceTime: "2021-08-01T00:00:00Z"
```

Comment:
might want to delete this section...

Reply:
@rmclaren thaks. I can change this with the query field, but to be consistent with @nicholasesposito I kept the same format and only updated the referenceTime. However, this does not calculate the correct dateTime.